diff -uraN watchgod-0.8.2/setup.py omv-watchgod-0.8.2/setup.py
--- watchgod-0.8.2/setup.py	2022-04-01 14:17:51.000000000 +0200
+++ omv-watchgod-0.8.2/setup.py	2025-12-26 20:01:29.591832778 +0100
@@ -47,7 +47,7 @@
     license='MIT',
     packages=['watchgod'],
     package_data={'watchgod': ['py.typed']},
-    install_requires=['anyio>=3.0.0,<4'],
+    install_requires=['anyio>=4.0.0,<5'],
     python_requires='>=3.7',
     zip_safe=True,
 )
